### PR TITLE
[hi] Add content/hi/docs/setup/best-practices/multiple-zones.md #29444

### PR DESCRIPTION
**Pull request by keshavcodex**
Localizes

- content/hi/docs/setup/best-practices/multiple-zones.md

This PR is in continuation of hi localization effort: #29353

/language hi

### CODE REVIEW

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+
+
+
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:5]:
```suggestion
```

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+title: कई क्षेत्रों में चलाना
+weight: 20
+content_type: concept
+---
+
+<!-- overview -->
+
+यह पृष्ठ कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करता है।
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:13]: Can you use more simpler wording for this `पृष्ठ कई क्षेत्रों` part?

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+<!-- body -->
+
+## पृष्ठभूमि
+
+कुबेरनेट्स को डिज़ाइन या गया है ताकि एक कुबेरनेट्स क्लस्टर कई विफलता क्षेत्रों में चल सके,
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:19]:
```suggestion
कुबेरनेट्स को डिज़ाइन किया गया है कि एक कुबेरनेट्स क्लस्टर कई विफलता क्षेत्रों में चल सके,
```

can you look for more simpler word than `विफलता`?

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+content_type: concept
+---
+
+<!-- overview -->
+
+यह पृष्ठ कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करता है।
+
+<!-- body -->
+
+## पृष्ठभूमि
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:17]:
```suggestion
## बैकग्राउंड
```

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+कुबेरनेट्स को डिज़ाइन या गया है ताकि एक कुबेरनेट्स क्लस्टर कई विफलता क्षेत्रों में चल सके,
+आमतौर पर जहां ये क्षेत्र फिट होते हैं
+एक तार्किक समूहन जिसे _क्षेत्र(region)_ कहा जाता है। प्रमुख क्लाउड प्रदाता एक क्षेत्र को परिभाषित करते हैं
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:21]:
```suggestion
एक तार्किक समूहन जिसे _region_ कहा जाता है। प्रमुख क्लाउड प्रदाता एक क्षेत्र को परिभाषित करते हैं
```

Over here, region is used a technical word.

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+विफलता क्षेत्रों के एक सेट के रूप में (जिसे _उपलब्धता क्षेत्र भी कहा जाता है_) जो
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:22]:
```suggestion
विफलता क्षेत्रों के एक सेट के रूप में (जिसे _availability zones_ कहा जाता है) जो
```

**Comment by mittalyashu** [comment]
@keshavcodex Can you go through the PR again and make sure the sentence expresses the correct meaning as mentioned in english docs?

**Comment by keshavcodex** [comment]
> @keshavcodex Can you go through the PR again and make sure the sentence expresses the correct meaning as mentioned in english docs?

ya sure.

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+उदाहरण के लिए, यदि आपका क्लाउड प्रदाता `टाइप = लोडबैलेंसर` के साथ सेवाओं का समर्थन करता है तो लोड बैलेंसर शायद केवल पॉड में चल रहे पॉड्स को ट्रैफिक भेज सकता है
+किसी दिए गए कनेक्शन को संसाधित करने वाले लोड बैलेंसर तत्व के समान क्षेत्र।
+विवरण के लिए अपने क्लाउड प्रदाता के दस्तावेज़ देखें।
+
+कस्टम या ऑन-प्रिमाइसेस डिप्लॉयमेंटस के लिए, समान विचार लागू होते हैं।
+{{< glossary_tooltip text="Service" term_id="service" >}} और
+{{< glossary_tooltip text="Ingress" term_id="ingress" >}} व्यवहार, जिसमें हैंडलिंग भी शामिल है
+अलग-अलग विफलता क्षेत्रों में, आपके क्लस्टर की स्थापना के तरीके के आधार पर भिन्न होता है।
+
+## Fault recovery
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:110]:
```suggestion
## Fault recovery

```

File: content/hi/docs/setup/best-practices/multiple-zones.md (hunk 2 above)
Comment (keshavcodex) [reply content/hi/docs/setup/best-practices/multiple-zones.md:13]: which one should i use 
- यह `पेज` कई `ज़ोन` में कुबेरनेट्स चलाने का वर्णन करती है।
- यह पृष्ठ कई `ज़ोन` में कुबेरनेट्स चलाने का वर्णन करती है।
- यह `पेज` कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करती है।

The english sentence was `This page describes running Kubernetes across multiple zones.`

File: content/hi/docs/setup/best-practices/multiple-zones.md (hunk 6 above)
Comment (sftim) [reply content/hi/docs/setup/best-practices/multiple-zones.md:22]: It's good to emphasize the equivalence / similarity between **zone** and **क्षेत्रों**.

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+आमतौर पर जहां ये क्षेत्र फिट होते हैं
+एक तार्किक समूहन जिसे _क्षेत्र(region)_ कहा जाता है। प्रमुख क्लाउड प्रदाता एक क्षेत्र को परिभाषित करते हैं
+विफलता क्षेत्रों के एक सेट के रूप में (जिसे _उपलब्धता क्षेत्र भी कहा जाता है_) जो
+सुविधाओं का एक सुसंगत सेट प्रदान करते हैं: एक क्षेत्र के भीतर, प्रत्येक क्षेत्र समान एपीआई और सेवाएं प्रदान करता है।
+
+विशिष्ट क्लाउड आर्किटेक्चर का उद्देश्य इस संभावना को कम करना है कि एक जोन कि विफलता दूसरे जोन में सेवाएं बाधित ना करे।
+
+## विमान नियंत्रण व्यवहार
+
+सभी [विमान नियंत्रण अवयव](/docs/concepts/overview/components/#विमान-नियंत्रण-अवयव)
```
Comment (sftim) [inline content/hi/docs/setup/best-practices/multiple-zones.md:29]:
```suggestion
सभी [विमान नियंत्रण अवयव](/docs/concepts/overview/components/#control-plane-components)
```
Take care not to translate the URLs of English pages.

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+कुबेरनेट्स एपीआई सर्वर के लिए क्रॉस-ज़ोन लचीलापन प्रदान नहीं करता है
+समापन बिंदु। उपलब्धता में सुधार के लिए आप विभिन्न तकनीकों का उपयोग कर सकते हैं
+क्लस्टर एपीआई सर्वर, जिसमें डीएनएस राउंड-रॉबिन, एसआरवी रिकॉर्ड, या
+स्वास्थ्य जांच के साथ एक तृतीय-पक्ष लोड संतुलन समाधान।
+{{< /note >}}
+
+## नोड व्यवहार
+
+कुबेरनेट्स स्वचालित रूप से पॉड्स को फैलाता है
+कार्यभार संसाधन (जैसे {{< glossary_tooltip text="डिप्लॉयमेंट(Deployment)" term_id="deployment" >}} or {{< glossary_tooltip text="स्टेटफुलसेट(StatefulSet)" term_id="statefulset" >}})
```
Comment (sftim) [inline content/hi/docs/setup/best-practices/multiple-zones.md:51]: How about:
```suggestion
कार्यभार संसाधन — जैसे {{< glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment" >}}(Deployment) 
या {{< glossary_tooltip text="स्टेटफुलसेट" term_id="statefulset" >}}(StatefulSet).
```
?

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+विफलता आपके पूरे कार्यभार को प्रभावित करती है।
+
+उदाहरण के लिए, आप यह सुनिश्चित करने के लिए एक बाधा सेट कर सकते हैं कि
+जब भी संभव हो, स्टेटफुलसेट की 3 प्रतिकृतियां अलग-अलग क्षेत्रों में चल रही हैं। आप इसे घोषणात्मक रूप से परिभाषित कर सकते हैं
+स्पष्ट रूप से परिभाषित किए बिना कि प्रत्येक वर्कलोड कौन से उपलब्धता क्षेत्र के उपयोग में हैं।
+
+### क्षेत्रों में नोड्स वितरित करना
+
+कुबेरनेट्स का कोर आपके लिए नोड नहीं बनाता है, आपको इसे स्वयं करने की आवश्यकता है,
+या अपनी ओर से नोड्स को प्रबंधित करने के लिए [क्लस्टर एपीआई](https://cluster-api.sigs.k8s.io/) जैसे टूल का उपयोग करें।
```
Comment (sftim) [inline content/hi/docs/setup/best-practices/multiple-zones.md:75]: I'd expect to see:
```suggestion
या अपनी ओर से नोड्स को प्रबंधित करने के लिए [Cluster API](https://cluster-api.sigs.k8s.io/) जैसे टूल का उपयोग करें।
```

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+{{< /note >}}
+
+## नोड व्यवहार
+
+कुबेरनेट्स स्वचालित रूप से पॉड्स को फैलाता है
+कार्यभार संसाधन (जैसे {{< glossary_tooltip text="डिप्लॉयमेंट(Deployment)" term_id="deployment" >}} or {{< glossary_tooltip text="स्टेटफुलसेट(StatefulSet)" term_id="statefulset" >}})
+एक क्लस्टर में विभिन्न नोड्स में। यह प्रसार मदद करता है
+विफलताओं के प्रभाव को कम करना।
+
+जब नोड शुरू होता है, तो प्रत्येक नोड पर क्यूबलेट स्वचालित रूप से नोड ऑब्जेक्ट के लिए {{< glossary_tooltip text="labels" term_id="label" >}} जोर देता है जो कुबेरनेट्स एपीआई में उस विशिष्ट क्यूबलेट का प्रतिनिधित्व करता है।
```
Comment (sftim) [inline content/hi/docs/setup/best-practices/multiple-zones.md:55]: Would
```suggestion
जब नोड शुरू होता है, तो प्रत्येक नोड पर क्यूबलेट स्वचालित रूप से नोड ऑब्जेक्ट के लिए {{< glossary_tooltip text="labelस" term_id="label" >}} जोर देता है जो कुबेरनेट्स एपीआई में उस विशिष्ट क्यूबलेट का प्रतिनिधित्व करता है।
```
work?

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+आपके द्वारा बनाए गए पॉड्स में लागू कर सकते हैं के साथ-साथ वर्कलोड संसाधनों में पॉड टेम्प्लेट के लिए जैसे डिप्लॉयमेंट, स्टेटफुलसेट, या नौकरी में लागू कर सकते हैं।
+
+## क्षेत्रों के लिए संग्रहण पहुंच
+
+जब लगातार वॉल्यूम बनाए जाते हैं, तो `PersistentVolumeLabel`
+[प्रवेश नियंत्रक](/docs/reference/access-authn-authz/admission-controllers/)
+स्वचालित रूप से किसी भी PersistentVolumes को ज़ोन लेबल में जोड़ता है जो किसी विशिष्ट क्षेत्र से जुड़े होते हैं। तब {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} सुनिश्चित करता है कि अपने `NoVolumeZoneConflict` विधेय के माध्यम से, वह पॉड्स जो किसी दिए गए PersistentVolume का दावा करते हैं
+केवल उस वॉल्यूम के समान क्षेत्र में रखे जाते हैं।
+
+आप एक {{< glossary_tooltip text="StorageClass" term_id="storage-class" >}} निर्दिष्ट कर सकते हैं
```
Comment (sftim) [inline content/hi/docs/setup/best-practices/multiple-zones.md:91]: If localizing Deployment to डिप्लॉयमेंट (see earlier) then I think we should be consistent and also transcribe StorageClass; these are both API kinds. Maybe add an explanation as well: something like (भंडारण का वर्ग).

File: content/hi/docs/setup/best-practices/multiple-zones.md (hunk 7 above)
Comment (sftim) [reply content/hi/docs/setup/best-practices/multiple-zones.md:110]: We should localize this heading.

**Review by sftim — COMMENTED** [review]
Some early feedback - I hope it's useful.

**Comment by keshavcodex** [comment]
> Some early feedback - I hope it's useful.

yeah i make all these changes that you have suggested

**Comment by mittalyashu** [comment]
@keshavcodex Let me know once you're made all the necessary changes?

File: content/hi/docs/setup/best-practices/multiple-zones.md (hunk 11 above)
Comment (keshavcodex) [reply content/hi/docs/setup/best-practices/multiple-zones.md:55]: it will confuse the reader, either `लेबलस` or `labels` will be better in my opinion

**Comment by keshavcodex** [comment]
> @keshavcodex Let me know once you're made all the necessary changes?

i have made all changes, should i push?

File: content/hi/docs/setup/best-practices/multiple-zones.md (hunk 11 above)
Comment (sftim) [reply content/hi/docs/setup/best-practices/multiple-zones.md:51]: “लेबलस (labels)” perhaps?

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 13]
```diff
@@ -0,0 +1,122 @@
+---
+
+
+
```
Comment (mittalyashu) [inline content/hi/docs/setup/best-practices/multiple-zones.md:4]:
```suggestion
```

**Comment by mittalyashu** [comment]
@keshavcodex We don't recommend copying and pasting the text as is because you're utilising a machine translator. Read the translated content carefully and make any required adjustments to verify that the grammar and meaning are correct.

Despite the fact that we're localising the documentation for Hindi speakers, not everyone is fluent in the language. We're keeping Hindi documents that will be read by people ranging in age from 16 to 70, so we have to compose and choose words carefully.

**Comment by keshavcodex** [comment]
> @keshavcodex We don't recommend copying and pasting the text as is because you're utilising a machine translator. Read the translated content carefully and make any required adjustments to verify that the grammar and meaning are correct.
> 
> Despite the fact that we're localising the documentation for Hindi speakers, not everyone is fluent in the language. We're keeping Hindi documents that will be read by people ranging in age from 16 to 70, so we have to compose and choose words carefully.

yes sir, i have read it many times and shifted many words left and right to make it sound grammatically correct in hindi, if there is still anything wrong, please say i will try to make it more simpler.

**Comment by keshavcodex** [comment]
yes sir, i have read it many times and shifted many words left and right to make it sound grammatically correct in hindi, if there is still anything wrong, please say i will try to make it more simpler. @mittalyashu

**Comment by keshavcodex** [comment]
Now its ready for review.

**Comment by keshavcodex** [comment]
It is ready for review

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/setup/best-practices/multiple-zones.md #29444

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+आपके क्लस्टर में कम से कम एक स्वस्थ नोड होने पर। उदाहरण के लिए: यदि सभी नोड्स
+अस्वस्थ हैं, आपको एक विशेष के साथ मरम्मत कार्य चलाने की आवश्यकता हो सकती है
+{{< glossary_tooltip text="टोलरेशन" term_id="toleration" >}} ताकि मरम्मत
+कम से कम एक नोड को सेवा में लाने के लिए पर्याप्त रूप से पूरा कर सकता है।
+
+कुबेरनेट्स इस चुनौती का कोई जवाब नहीं लेकर आया; हालांकि, यह विचार करने के लिए कुछ है।
+
+## {{% heading "whatsnext" %}}
+
+यह जानने के लिए कि शेड्यूलर पॉड्स को क्लस्टर में कैसे रखता है, कॉन्फ़िगर की गई बाधाओं का सम्मान कैसे करता है, इसे देखें [शेड्यूलिंग और बेदखली](/docs/concepts/scheduling-eviction/)।
```
Comment (jayesh-srivastava) [inline content/hi/docs/setup/best-practices/multiple-zones.md:92]: `यह जानने के लिए कि शेड्यूलर पॉड्स को क्लस्टर में कैसे रखता है, कॉन्फ़िगर की गई बाधाओं का सम्मान कैसे करता है, [शेड्यूलिंग और बेदखली](/docs/concepts/scheduling-eviction/) को देखें ।`

File: content/hi/docs/setup/best-practices/multiple-zones.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+
+आप [नोड चयनकर्ता बाधाओं](/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) को आपके द्वारा बनाए गए पॉड्स में लागू कर सकते हैं, साथ-साथ वर्कलोड संसाधनों में पॉड टेम्प्लेट के लिए जैसे डिप्लॉयमेंट, स्टेटफुलसेट, या नौकरी में भी लागू कर सकते हैं।
+
+## क्षेत्रों के लिए स्टोरेज एक्सेस
+
+जब परसिस्टेंट वॉल्यूम बनाए जाते हैं, तो `PersistentVolumeLabel`
+[प्रवेश नियंत्रक](/docs/reference/access-authn-authz/admission-controllers/)
+स्वचालित रूप से किसी भी परसिस्टेंट वॉल्यूम को ज़ोन लेबल में जोड़ता है जो किसी विशिष्ट क्षेत्र से जुड़े होते हैं। तब {{< glossary_tooltip text="शेड्यूलर" term_id="kube-scheduler" >}} सुनिश्चित करता है कि अपने `NoVolumeZoneConflict` विधेय के माध्यम से, वह पॉड्स जो किसी दिए गए परसिस्टेंट वॉल्यूम का दावा करते हैं केवल उस वॉल्यूम के समान क्षेत्र में रखे जाते हैं।
+
+आप एक परसिस्टेंट वॉल्यूम कलेम्स के लिए {{< glossary_tooltip text="भंडारण का वर्ग" term_id="storage-class" >}} निर्दिष्ट कर सकते हैं जो असफल डोमेन (ज़ोन) को निर्दिष्ट करता है कि उस वर्ग में भंडारण का उपयोग कर सकते हैं।
```
Comment (jayesh-srivastava) [inline content/hi/docs/setup/best-practices/multiple-zones.md:64]: It should be `क्लेम्स` instead of `कलेम्स`.

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

**Comment by divya-mohan0209** [comment]
@keshavcodex : Please could you check and advise whether you're ready to work on the changes mentioned?

**Comment by keshavcodex** [comment]
i am sorry but i won't be able to continue this work

**Pull request closed without merging**